### PR TITLE
Add configurable bind address

### PR DESCRIPTION
**Pull request by tituspijean**
Adds ability to configure which address and port the app will be bound.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -24,10 +24,12 @@ var (
 func init() {
 	// command-line flags
 	flagDisableLogin := flag.Bool("disable-login", false, "Disable login page. Turn off authentication.")
+        flagBindAddress := flag.String("bind-address", "0.0.0.0:5000", "Address:Port to which the app will be bound.")
```
Comment (ngoduykhanh) [inline main.go:27]: Can you adjust the code indentation?

File: main.go [hunk 2]
```diff
@@ -37,6 +39,7 @@ func init() {
 	fmt.Println("Build Time\t:", buildTime)
 	fmt.Println("Git Repo\t:", "https://github.com/ngoduykhanh/wireguard-ui")
 	fmt.Println("Authentication\t:", !util.DisableLogin)
+        fmt.Println("Bind address\t:", util.BindAddress)
```
Comment (ngoduykhanh) [inline main.go:42]: here as well.

File: go.mod [hunk 3]
```diff
@@ -12,6 +12,8 @@ require (
 	github.com/labstack/echo/v4 v4.1.16
 	github.com/labstack/gommon v0.3.0
 	github.com/leodido/go-urn v1.2.0 // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v1.0.1 // indirect
```
Comment (ngoduykhanh) [inline go.mod:16]: Are they used anywhere?

Comment (tituspijean) [reply go.mod:16]: They got automatically added when I followed your steps to add assets and compile. I am not fluent enough in Go to know if that's important. :/

File: main.go (hunk 1 above)
Comment (tituspijean) [reply main.go:27]: Uh weird. I used tabs though. I am on mobile right now, I will fix that later today. ;)

File: go.mod (hunk 3 above)
Comment (ngoduykhanh) [reply go.mod:16]: As far as I can see it is not used anywhere so you can remove them :)

**Comment by tituspijean** [comment]
We should be good to go now. :)

**Comment by tituspijean** [comment]
Uh, no, actually. Routes seem hardcoded and do not handle well a proxy on something like `domain.tld/wireguard`. Everything gets linked to `domain.tld/`, ignoring the subpath.

**Comment by ngoduykhanh** [comment]
> Uh, no, actually. Routes seem hardcoded and do not handle well a proxy on something like `domain.tld/wireguard`. Everything gets linked to `domain.tld/`, ignoring the subpath.

Usually, it should be handled in another reverse proxy layer, such as Nginx. Feel free to open an issue if you see it have a problem with the code. 

Btw, it is not related to the PR :)

**Comment by ngoduykhanh** [comment]
Merged. Thanks for your contribution.

**Pull request merged**